### DISCUSSION TITLE
Panasonic Satelittenfrequenz in Senderliste ändern

### DISCUSSION THREAD

**Discussion by deepblackman** [PredatH0r/ChanSort#392] [Q&A]
Habe am Unicable-System 3 Panasonic-Geräte (2x Fernsehen, 1x Recorder). An einem Fernseh gibt es nach dem Suchlauf für Senderliste Satelittensender die gefunden werden, aber dann kein Bild und Ton haben, bei den zwei anderen Geräten aber funktionieren. Habe ChanSort festgestellt, das die MHZ-Frequenz des Senders immer etwas verscheiden sind zwischen funktionierenden Geräten und nicht funktionierendem Gerät. z.B. hr-fernsehen HD 10893 mHz (funktioniert) 10886 mHz bei Fernseher wo es nicht funktioniert. Verstehe es nicht, wieso obwohl Geräte am gleichen Unicabel-System hängen hier verschiedene MHz nach Sendersuchlauf haben. Leider kann man an den Panasonic-Geräten die Frequenz für einen Sender nicht manuell eingeben und auch ChanSort geht es nicht. Auch habe ich keine Software gefunden, womit ich dies ändern könnte. Gibt es da eine Idee?

**Comment by PredatH0r** [comment]
7 MHz Unterschied ist schon massiv. Bei DVB-C reichen 8 MHz für einen kompletten Transponder mit mehreren Sendern (mit Symbolrate 6900 kS/s).

Die Frequenz ist nicht spezifisch für einen Sender, sondern immer für einen ganzen Transponder, auf dessen Frequenz mehrere Sender in einen gemeinsamen Datenstrom zusammengemischt werden (MUX). 
Man kann deshalb bei der Sendersuche oft "Manuell" auswählen, eine Transponderfrequenz eingeben und dann werden alle Sender auf dieser Frequenz gefunden.

In ChanSort werde ich aus unzähligen Gründen nie die Bearbeitung von Tunerdaten einbauen. Das ist viel zu spezifisch für das geweilige Gerät und erfordert eine Unmenge an Daten, die weder ich noch Anwender kennen und viel zu riskant wären. Nur die Firmware des Geräts kann zuverlässig die nötigen Daten beim Sendersuchlauf ermitteln und speichern.

Je nach Modell bzw. Serie verwendet Panasonic unterschiedliche Hard- und Firmware von verschiedenen Zulieferern. Die diversen Plattformen können sich also durchaus unterschiedlich verhalten und exportieren auch verschiedene Senderlistendateiformate.

Bei "svl.db" und "svl.bin" könnte man, wenn man experimentierfreudig ist, die Datei mit ChanSort öffnen. Dabei wird eine entschlüsselte Version der Datei im gleichen Ordner abgespeichert. Diese SqlLite-Datenbankdatei könnte man dann mit einem Tool wie "SQLite Studio" oder "DB Browser for SQLite" öffnen und in der Tabelle "TSL" in den Daten den Wert in Spalte "freq" ändern.
Danach in ChanSort auf "Speichern" klicken, damit die verschlüsselte Datei aktualisiert wird.
Ich selber habe dies aber noch nie versucht, da ich kein solches Panasonic Gerät besitze.

![grafik](https://github.com/PredatH0r/ChanSort/assets/6238104/5150c414-9dd1-4e72-8930-e8768ca8b9c8)
